Python_Scripts/compareStateTrajectories_BioModels_1.py

```diff
@@ -38,66 +30,79 @@ def compStaTraj_BioModels():
+                    sbml_path = os.path.join(
+                        DIR_MODELS_SEDML, iModel, 'sbml_models',
+                        iFile + '.sbml')
 
                     # Get whole model
-                    model = all_settings(iModel, iFile, 1)
+                    try:
+                        model = all_settings(iModel, iFile)
+                    except Exception as e:
+                        print(f'Model {iModel} failed ', e)
+                        continue
 
                     ######### COPASI simulatiom
                     # open new .csv file with COPASI simulation trajectories
-                    try:
-                        tsv_file = pd.read_csv(tsv_path + '/Original_COPASI_' + iModel + '_14_14.tsv', sep='\t')
-                    except:
-                        tsv_file = pd.read_csv(tsv_path + '/Original_COPASI_' + iModel + '_12_12.tsv', sep='\t')
+                    if os.path.exists(os.path.join(
+                            DIR_MODELS_TRAJ_REF_BIOMODELS,
+                            f'Original_COPASI_{iModel}_14_14.tsv')):
+                        tsv_file = pd.read_csv(os.path.join(
+                            DIR_MODELS_TRAJ_REF_BIOMODELS,
+                            f'Original_COPASI_{iModel}_14_14.tsv'),
+                            sep='\t')
+                    else:
+                        # TODO Y Why the alternative?
+                        tsv_file = pd.read_csv(os.path.join(
+                            DIR_MODELS_TRAJ_REF_BIOMODELS,
+                            f'Original_COPASI_{iModel}_12_12.tsv'),
+                            sep='\t')
 
                     # columns names of .tsv file and alter .tsv file
                     column_names = list(tsv_file.columns)
                     column_names.remove('# Time')
                     column_names = column_names[:len(column_names) - 1]
                     position = []
+
+                    # TODO Y ????
                     del_counter = 0
                     for iName in range(0, len(column_names)):
                         if 'Values[' in column_names[iName - del_counter]:
-                            column_names.remove(column_names[iName - del_counter])
+                            column_names.remove(
+                                column_names[iName - del_counter])
                             position.append(iName)
                             del_counter += 1
                     del tsv_file['# Time']
-                    tsv_file.drop(tsv_file.columns[len(tsv_file.columns) - 1], axis=1, inplace=True)
+                    tsv_file.drop(
+                        tsv_file.columns[len(tsv_file.columns) - 1],
+                        axis=1, inplace=True)
                     if len(position) != 0:
-                        tsv_file = tsv_file.drop(tsv_file.columns[position], axis=1)
+                        tsv_file = tsv_file.drop(
+                            tsv_file.columns[position], axis=1)
```

Comment:
This whole part looks somewaht fishy and like something we want to redo...

Reply:
Yes. Best: Completely remove the compareStateTrajectories_BioModels_1.py aas ...JWS_1 is neeearly the same (also :fish: y though)